cpp/src/platform/windows/LogImpl.cpp

```diff
@@ -41,7 +41,7 @@ errno_t fopen_s(FILE** pFile, const char *filename, const char *mode)
 {
     if (!pFile)
     {
-        _set_errno(EINVAL);
```

Comment:
Thanks, I'll try this approach. By the way, what's the point of using _set_errno ?